.github/workflows/publish.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CGO_ENABLED: 0
```

Comment:
Isn't it also important to add this flag for the Windows version?

Reply:
I'm not sure and since i do not have a way to test it. I'd like to be safe WDYT ?

Reply:
Running the build on my fork to ensure that it works: https://github.com/johnsudaar/cli/actions/runs/3855819102